docs/build/user-consent.md

```diff
@@ -158,7 +158,12 @@ try await client.contacts.deny(addresses: [spamConvo.peerAddress, unwantedConvo.
 </TabItem>
 <TabItem value="dart" label="Dart"  attributes={{className: "dart_tab"}}>
 
-The user consent feature for Dart hasn't been implemented yet
+```dart
+var auth = AuthManager(client.address, api);
+
+await contacts.deny(auth.keys, wantedConvo.address);
+await contacts.allow(auth.keys, spamConvo.address);
+```
```

Comment:
I think instead we should suggest the high-level recipe:
```dart
await client.allowContact(wantedConvo.address);
await client.denyContact(spamConvo.address);
```

Reply:
Ah yes! Found the higher-level testing file. Updated snippets